orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/HttpClientUtils.java

```diff
@@ -77,6 +78,13 @@ public long getRetryInterval() {
       }
     );
 
+    httpClientBuilder.setRetryHandler((exception, executionCount, context) -> {
+      HttpUriRequest currentReq = (HttpUriRequest) context.getAttribute(HttpCoreContext.HTTP_REQUEST);
+      Uninterruptibles.sleepUninterruptibly(RETRY_INTERVAL, TimeUnit.MILLISECONDS);
```

Comment:
do we want to verify the type of exception here?

I'd think we would only retry on some subclass of  IOException